src/System.Security.Cryptography.Csp/tests/RSAImportExportCspBlobTests.cs

```diff
@@ -37,7 +37,7 @@ public static void ExportImportPublicOnly()
 
                 Assert.Equal(exported, exported2);
 
-                Assert.Throws<CryptographicException>(() => rsa.ExportCspBlob(includePrivateParameters: true));
+                Assert.ThrowsAny<CryptographicException>(() => rsa.ExportCspBlob(includePrivateParameters: true));
```

Comment:
If Windows was throwing a CryptographicException before, why is it changing to a WindowsCryptographicException now?